nerfstudio/process_data/colmap_utils.py

```diff
@@ -455,10 +463,12 @@ def colmap_to_json(
         if image_id_to_depth_path is not None:
             depth_path = image_id_to_depth_path[im_id]
             frame["depth_file_path"] = str(depth_path.relative_to(depth_path.parent.parent))
+
+        if use_single_camera_mode is True:  # add the camera for this frame
```

Comment:
Should this be False?

Reply:
Just fixed this in https://github.com/nerfstudio-project/nerfstudio/pull/3081/commits/999b7617ce833ead75ab2c553c1bc2231c0e1215